Dockerfile

```diff
@@ -30,18 +25,29 @@ RUN set -x \
     && docker-php-ext-install odbc \
     && docker-php-source delete
 
-COPY --from=0 /tmp/snowflake-odbc.deb /tmp/snowflake-odbc.deb
-RUN dpkg -i /tmp/snowflake-odbc.deb
-ADD ./driver/simba.snowflake.ini /usr/lib/snowflake/odbc/lib/simba.snowflake.ini
+#snoflake download + verify package
+ADD driver/snowflake-policy.pol /etc/debsig/policies/$SNOWFLAKE_GPG_KEY/generic.pol
```

Comment:
Tady by měl být COPY https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
jo, dobry postreh, mame to tak aj inde: https://github.com/keboola/provisioning-router/blob/22f70258c6f63c52dc889118d53da44e328d76d1/Dockerfile#L27